eclipse/src/saros/ui/widgets/viewer/session/XMPPSessionDisplayComposite.java

```diff
@@ -33,33 +28,13 @@
  */
 public final class XMPPSessionDisplayComposite extends SessionDisplayComposite {
 
-  private static final Logger LOG = Logger.getLogger(XMPPSessionDisplayComposite.class);
-
-  @Inject private ConnectionHandler connectionHandler;
   @Inject private XMPPContactsService contactsService;
 
-  private final IConnectionStateListener connectionListener =
-      (state, error) -> {
-        if (state == ConnectionState.CONNECTED || state == ConnectionState.NOT_CONNECTED) {
```

Comment:
Why is this listener now unnecessary? It seems pretty useful to expand the view when a new connection gets established. (Expanding it on disconnection however is not that useful as.)

This does not seem directly connected to the other part of the PR as that logic only expands the view when a new contact is added. Is there another listener that already expands the contacts view when a new connection is established?

Reply:
Because it just works for the disconnect case and this case is not needed.

The contact list is de facto empty after connection start, and existing contacts on the roster are added via roster / ContactsService listener updates as `UpdateType.ADDED` in `SessionContentProvider` leading to expanding the contact list right after start.

So this deleted code has no implication and if a user decides to collapse the contact list in the mean time I don't think it should be open at disconnect.